README.md

```diff
@@ -1,6 +1,6 @@
 <p align="center">
-  <a href="https://risczero.com/#gh-dark-mode-only"><img src="docs/assets/RISC_Zero_Logo_darkmode.png" height="100"></a>
-  <a href="https://risczero.com/#gh-light-mode-only"><img src="docs/assets/RISC_Zero_Logo_lightmode.png" height="100"></a>
+  <a href="https://risczero.com/#gh-dark-mode-only"><img src="assets/RISC_Zero_Logo_darkmode.png" height="100"></a>
+  <a href="https://risczero.com/#gh-light-mode-only"><img src="assets/RISC_Zero_Logo_lightmode.png" height="100"></a>
```

Comment:
I think we should keep the logo assets in the docs folder, for 2 reasons:
* These are used for docs (namely, the main readme)
* It reduces the number of top-level folders